modules/apps/style-book/style-book-api/src/main/java/com/liferay/style/book/model/StyleBookEntryModel.java

```diff
@@ -303,6 +303,37 @@ public interface StyleBookEntryModel
 	 */
 	public void setFrontendTokensValues(String frontendTokensValues);
 
+	/**
+	 * Returns the frontend token definition ID of this style book entry.
+	 *
+	 * @return the frontend token definition ID of this style book entry
+	 */
+	@AutoEscape
```

Comment:
I think these changes should be auto generated by build service, so they should be moved to that commit. Make sure all changes that are made by running a command are in a separate commit.